pages/windows/scoop-bucket.md

```diff
@@ -0,0 +1,25 @@
+# scoop bucket
+
+> A bucket is an application repository.
+> Some buckets are already known to scoop, so you don't have to provide an url.
```

Comment:
Is this better?
```suggestion
> if Scoop doesn't know where the bucket is located we should specify its repository location.
```

Reply:
Yeah this also does sound a lot better. Just one small remark: Maybe we can replace the `should` with a `have to`. By using `should` this does sound more like a suggestion, rather than an instruction.